appsettings/src/tree/AppSettingsTreeItem.ts

```diff
@@ -52,21 +52,26 @@ interface AppSettingsTreeItemOptions {
 
 export class AppSettingsTreeItem extends AzExtParentTreeItem {
     public static contextValue: string = 'applicationSettings';
-    public readonly label: string = 'Application Settings';
+    public label: string = 'Application Settings';
```

Comment:
localize this too

Reply:
For consistency's sake

Reply:
![200w](https://github.com/user-attachments/assets/7b6a13b5-f80b-4164-8edb-78ef1cde6824)